<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextBrowser" name="msgBrowser">
           <property name="minimumSize">
            <size>
             <width>533</width>
             <height>241</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>533</width>
             <height>241</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFontComboBox" name="fontCbx">
             <property name="minimumSize">
              <size>
               <width>188</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sizeCbx">
             <property name="minimumSize">
              <size>
               <width>69</width>
               <height>20</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>17</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>21</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>22</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="boldTBtn">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/res/bold.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <selectedoff>:/images/bold.png</selectedoff>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="italicTBtn">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/res/italic.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="underlineTBtn">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/res/under.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="colorTBtn">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/res/color.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="sendTBtn">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/res/send.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/send.png</normaloff>:/images/send.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="saveTBtn">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/res/save.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="clearTBtn">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/res/clear.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="msgTextEdit">
           <property name="minimumSize">
            <size>
             <width>533</width>
             <height>95</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>533</width>
             <height>95</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="usrTblWidget">
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>IP地址</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sendBtn">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="usrNumLbl">
         <property name="text">
          <string>在线用户：0人</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="AudioBtn">
         <property name="text">
          <string>开始录音</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exitBtn">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
